libethcore/ICAP.cpp

```diff
@@ -111,7 +111,19 @@ std::string ICAP::encoded() const
 {
 	if (m_type == Direct)
 	{
+#ifdef __INTEL_COMPILER
```

Comment:
So what is happening here?   Why is Intel needing an entirely different code-path?